ui/app/components/aoi/SearchAOIToolbar.js

```diff
@@ -55,8 +58,28 @@ export class SearchAOIToolbar extends Component {
         }
       });
     }
-    // If 2 or more characters are entered then make request for suggested names.
-    if (e.length >= 2) {
+    const matchISO3 = e.split(":")
+    if (matchISO3.length === 2 && matchISO3[1].length === 3) {
```

Comment:
It would be better to check if it starts with `code:` and has other 3 letters

![image](https://user-images.githubusercontent.com/666291/90688091-4f0b1000-e244-11ea-86f8-933163701092.png)